src/core/evaluator.js

```diff
@@ -971,7 +972,9 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         for (var i = 0; i < glyphs.length; i++) {
           var glyph = glyphs[i];
           if (!glyph) { // Previous glyph was a space.
-            width += textState.wordSpacing * textState.textHScale;
+            var sp_tx = textState.wordSpacing * textState.textHScale;
```

Comment:
Nit: please remember to use more descriptive (and complete) variable names :-)
Also don't forget that variables should be lowerCamelCase, for reference: https://github.com/mozilla/pdf.js/wiki/Style-Guide#naming.